code/game/turfs/turf.dm

```diff
@@ -498,10 +498,6 @@
 	return !slayer && ..()
 
 
-/turf/open/floor/plating/plating_catwalk/is_weedable() //covered catwalks are unweedable
-	. = ..()
-	if(covered)
-		return FALSE
```

Comment:
Not sure if this is good, have you seen what it looks like on prison or the mainship?

Reply:
I think part of my catwalk PR got mixed up in this. Lemme cut those out.